ms/MSOper/MSMetaData.h

```diff
@@ -769,6 +769,7 @@ class MSMetaData {
     mutable std::map<ScanKey, std::set<Int> > _scanToStatesMap, _scanToFieldsMap, _scanToAntennasMap;
     mutable std::map<Int, std::set<Int> >    _fieldToStatesMap, _stateToFieldsMap, _sourceToFieldsMap;
     mutable std::map<std::pair<uInt, uInt>, uInt> _spwPolIDToDataDescIDMap;
+    mutable std::vector<std::vector<uInt>> _spwIDToPolIDMap;
```

Comment:
The name implies that this is a map, why not use an `std::map` instead?

Reply:
The spw id is represented by the index of the (outer) vector. This is a common pattern used in this class, and often the variable name includes "Map" which refers the the index being one side of the map.